.travis.yml

```diff
@@ -1,5 +1,6 @@
 language: c
 cache: ccache
+dist: trusty
```

Comment:
dot-asm, LMGTFY again... turns out you can unroll the packages in the matrix:
https://github.com/travis-ci/travis-ci/issues/3505#issuecomment-91057872

Want to try and send a pull request for that?